Exercise_F.js

```diff
@@ -10,6 +10,12 @@
 
 function occurrence(string, targetChar) {
   // CODE HERE
+  let x = string.indexOf(targetChar);
+  let a = 0
+  while (x > -1){
+    string = string.replace(targetChar, "");
+    x = string.indexOf(targetChar);
+    a++
+  }
+  return a;

```

Comment:
Interesting logic to achieve this answer it works well!
By nature clean code is readable and easy to understand, consider renaming your variable names to something that describes the information that they contain